docs/11_view.md

```diff
@@ -0,0 +1,85 @@
+# View Dandisets
+
+## Browse `Dandisets`
+
+When you go the [DANDI Web application](https://dandiarchive.org/) you can click 
+on `PUBLIC DANDISET` to have access to all `Dandisets` currently available 
+in the archive and you can sort them  by name, identifier or date of modification. 
```

Comment:
in the archive and you can sort them by name, identifier, or date of modification.